src/pages/NewResearcherConsole.js

```diff
@@ -141,6 +141,27 @@ export default function NewResearcherConsole(props) {
     }
   };
 
+  //resubmit collection function, passed to collections table to be used in buttons
+  const resubmitCollection = async (darCollection) => {
+    try {
+      const { darCollectionId, darCode } = darCollection;
+      const resubmittedCollection = await Collections.resubmitCollection(darCollectionId);
+      const targetIndex = researcherCollections.findIndex((collection) =>
+        collection.darCollectionId === darCollectionId);
+      if (targetIndex < 0) {
+        throw new Error("Error: Could not find target collection");
+      }
+      const clonedCollections = cloneDeep(researcherCollections);
+      clonedCollections[targetIndex] = resubmittedCollection;
+      setResearcherCollections(clonedCollections);
+      Notifications.showSuccess({text: `Revising collection ${darCode}`});
+    } catch (error) {
+      Notifications.showError({
+        text: 'Error: Cannot revise target collection'
+      });
+    }
+  };
+
```

Comment:
Somehow, I'm seeing "Revise" for DARs that are under election which should be incorrect. Will dig in to see if I can spot the issue.
![Screen Shot 2022-01-18 at 8 30 00 AM](https://user-images.githubusercontent.com/116679/149946662-c006045b-2ff0-4b29-8449-f76e4de61d63.png)

Reply:
OK, the error case here is not the "Revise" button, but the election status. In the case of `DAR-252`, it is `Canceled`, not `Under Election`
![Screen Shot 2022-01-18 at 8 33 22 AM](https://user-images.githubusercontent.com/116679/149947169-2d26d9ad-c2c4-4ff0-af27-1af28c34a80c.png)